docs/DeveloperGuide.md

```diff
@@ -484,6 +588,7 @@ _Non-functional requirements specify the constraints under which App-Ointment is
 * **Appointment Schedule**: The list of appointments maintained in App-Ointment, arranged by appointment datetime.
 * **Patient Records**: The list of patients maintained in App-Ointment.
 * **Doctor Records**: The list of doctors maintained in App-Ointment.
+* **Appointment Records**: The list of Appointment maintained in App-Ointment.
```

Comment:
isn't this already mentioned under `Appointment Schedule` at line 588? I don't think we're using the term Appointment Records at all

Reply:
oh i see, i saw wrongly